kroxylicious-operator/src/main/resources/META-INF/fabric8/virtualkafkaclusters.kroxylicious.io-v1.yml

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+# Note the v1 in the filename refers to the version of the CustomResourceDefinition
+# not any of the versions of API being defined.
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  # name must match the spec fields below, and be in the form: <plural>.<group>
+  name: virtualkafkaclusters.kroxylicious.io
+spec:
+  # group name to use for REST API: /apis/<group>/<version>
+  group: kroxylicious.io
+  scope: Namespaced
+  names:
+    plural: virtualkafkaclusters
+    singular: virtualkafkacluster
+    kind: VirtualKafkaCluster
+    shortNames:
+      - vkc
+  # list of versions supported by this CustomResourceDefinition
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      subresources:
+        status: { }
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            metadata:
+              type: object
+            spec:
+              type: object
+              required: ["proxyRef", "targetCluster"]
+              properties:
+                proxyRef:
+                  type: object
+                  required: [ "name" ]
+                  properties:
+                    group:
+                      type: string
+                      maxLength: 253
+                      pattern: ^$|^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$
+                    kind:
+                      type: string
+                      maxLength: 63
+                      minLength: 1
+                      pattern: ^[a-zA-Z]([-a-zA-Z0-9]*[a-zA-Z0-9])?$
+                    name:
+                      maxLength: 253
+                      minLength: 1
+                      type: string
+                targetCluster:
+                  type: object
+                  required: [ "bootstrapping" ]
+                  properties:
+                    bootstrapping:
```

Comment:
`bootstrapAddress`

Reply:
Do you mean the `bootstrapping` or `bootstrapping.bootstrap`. I've renamed `bootstrapping.bootstrap` to `bootstrapping.bootstrapAddress`. We still want the `bootstrapping` object to add more pieces to it like the nodeId ranges.